react/modules/gaEvents.ts

```diff
@@ -118,3 +119,36 @@ export function selectPromotion(eventData: PixelMessage['data']) {
 
   updateEcommerce(eventName, data)
 }
+
+export function addToCart(eventData: PixelMessage['data']) {
+  if (!shouldMergeUAEvents()) return
+
+  const eventName = 'add_to_cart'
+
+  const {
+    currency,
+    items: [item],
```

Comment:
Here we're retrieving only the first item of `items`, but there could be more than 1. 

I also thought there could only be 1, but after some research, I found out there are some cases where more products are added in the same event! 🤯 There's this ["Buy Together"](https://github.com/vtex-apps/recommendation-shelf#:~:text=recommendation%2Dshelf-,recommendation%2Dbuy%2Dtogether,-recommendation%2Drefresh) component that lets [multiple products to be added at once](https://github.com/vtex-apps/add-to-cart-button/blob/8b31f7189f567c5e088d1f7592a0b690abd69629/react/AddToCartButton.tsx#L207-L221).
![](https://user-images.githubusercontent.com/20840671/87463820-712bc580-c5e8-11ea-80f8-e773f3a61f7e.gif)

Can we update this function to deal with multiple items?